R/render.R

```diff
@@ -188,6 +178,16 @@ render <- function(path = ".",
 
 }
 
+site_url <- function(config_contents, type) {
+
+  if (nzchar(Sys.getenv("BABELQUARTO_CI_URL"))) {
+    return(Sys.getenv("BABELQUARTO_CI_URL"))
+  }
+
+  config_contents[[type]][["site-url"]] %||% ""
```

Comment:
I really can't understand why it was more complicated but I need to re-read the PR.

Related https://github.com/ropensci-review-tools/babelquarto/pull/9

Reply:
But what you currently do on CI does not depend on previous behavior?

Reply:
I don't understand your question. I had to set up the BABELQUARTO_CI_URL env variable for my CI in my [babelbook](https://github.com/Nenuial/babelbook) to work.

I now tried it without the env var and this PR, and it works fine.